tests/test_web_urldispatcher.py

```diff
@@ -458,3 +458,18 @@ async def test_static_absolute_url(aiohttp_client: Any, tmp_path: Any) -> None:
     client = await aiohttp_client(app)
     resp = await client.get("/static/" + str(file_path.resolve()))
     assert resp.status == 403
+
+
+async def test_for_issue_5250(aiohttp_client: Any, tmp_path: Any) -> None:
+    app = web.Application()
+    app.router.add_static("/foo", tmp_path)
+
+    async def get_foobar(request):
+        return web.Response(body="success!")
+
+    app.router.add_get("/foobar", get_foobar)
```

Comment:
I feel like there should be more tested cases and maybe some corner cases identified.